cremalab/scripts
# GH-168: Use `HYGEN_TMPLS` instead of symlinks

## Description

See #168 

e876d0e GH-168: Update cremalab/app-mobile ref
e1685ba GH-168: Update cremalab/app-web ref
f87e544 GH-168: Use HYGEN_TMPLS instead of symlinks for @cremalab/scripts-web
e2eff68 GH-168: Use HYGEN_TMPLS instead of symlinks for @cremalab/scripts-mobile
db0a8ca GH-168: Use HYGEN_TMPLS instead of symlinks for @cremalab/scripts

closes #168 

### 📋 Acceptance Criteria

> _Checked off by PR **Assignees**_

- [x] Everything works the same
- [x] A `_templates` directory does not temporarily show up in the consumer



### 👩‍🔬 Test Instructions

> _Provided by PR **Assignees**_

1. `npm run dev:setup`
2. Navigate to consumers and run all scripts: `new:(type|util|component)` and make sure generators work
3. Make sure that you never see a `_templates` directory pop into the consumer directory

## 🔎 Reviewer Checklist

> _Checked off by PR **Reviewers**_




---

> ### Roles & Responsibilities
>
> #### 👨‍💻 Assignee
>
> - Initiator of this PR (be sure to set in GitHub UI)
> - Addresses feedback and change requests
> - Merges PR once approved (usually deletes branch unless `develop` or `release`)
>
> #### 👩‍💻 Reviewer
>
> - Invited to review PR by **Assignee** (via GitHub UI)
> - Is expected to complete a review and address followup
